rts/Sim/Units/CommandAI/BuilderCaches.cpp

```diff
@@ -0,0 +1,147 @@
+#include "Sim/Units/CommandAI/BuilderCaches.h"
+#include "Sim/Misc/TeamHandler.h"
+#include "Sim/Units/UnitHandler.h"
+#include "Sim/Units/Unit.h"
+
+// not adding to members, should repopulate itself
```

Comment:
Previously it was obvious what "members" were because this chunk was right under a bunch of creg registrations
```suggestion
// not adding to creg, should repopulate itself
```